org.sil.Bloom.yml

```diff
@@ -0,0 +1,647 @@
+                  sources:
+                      - type: archive
+                        url: https://www.python.org/ftp/python/2.7.18/Python-2.7.18.tar.xz
+                        sha256: b62c0e7937551d0cc02b8fd5cb0f544f9405bafc9a54d3808ed4594812edef43
+          - name: gtk-sharp3
+            sources:
+                - type: git
+                  url: "https://github.com/mono/gtk-sharp.git"
+                  commit: 9ef7d3f394b9d485674c59e92db24ac66fd9f16b
+                - type: patch
+                  #https://github.com/scx/org.gnome.gbrainy/blob/ed2883a89cfd996c8e1e2a83ff23586c8971a0b6/gtk-sharp-profiler-startup.patch
+                  path: patches/gtk-sharp3/gtk-sharp-profiler-startup.patch
+          - name: mono5-sil-msbuild
+            # It was challenging to build msbuild from source. Other projects
+            # appear to fetch binaries rather than build msbuild. Following that.
+            buildsystem: simple
+            build-commands:
+                # Extract opt/mono5-sil/* into /app/
+                - |
+                    for package in *.deb; do
+                      mkdir "${package}-dir" &&
+                        (cd "${package}-dir" && ar x ../"${package}") &&
+                        tar -C /app -x -f "${package}-dir"/data.tar.xz \
+                          --strip-components=3 ./opt/mono5-sil
+                    done
+                # Use same-character-count path to /app since some binary files are
+                # going to be modified too.
+                - perl -pi -e 's#/opt/mono5-sil#/./././././app#g' /app/bin/*
+            cleanup:
+                # Can't clean '*' because that seems to result in /app/bin/mono being
+                # absent.
+                - "/lib/mono/msbuild"
+                - "/lib/mono/xbuild"
+            sources:
+                - type: file
+                  url: http://linux.lsdev.sil.org/ubuntu/pool/main/m/mono5-sil-msbuild/mono5-sil-msbuild_15.8+xamarinxplat.2018.07.31.22.43-0xamarin5+ubuntu1804b1_all.deb
+                  sha256: 758274d8bc438f325fc5fc808c0482ab50eb2bdaf02f61747888e190760753de
+          - name: libgdiplus
+            build-options:
+                prefix: /app
+                env:
+                    MONO_PREFIX: /app
+            config-opts:
+                - --with-pango
+            cleanup:
+                - "/include"
+                - "*.a"
+            sources:
+                - type: git
+                  url: https://github.com/sillsdev/libgdiplus
+                  tag: merge/5.6.0
+                  commit: b9c581ba87da89651d5ec56c5da0960ab739a63a
+            modules:
+                - name: giflib
+                  buildsystem: simple
+                  build-commands:
+                      # Skip building documentation
+                      - perl -pi -e 's#doc/Makefile##' configure.ac
+                      - ./autogen.sh --prefix=/app
+                      - make
+                      - make install
```

Comment:
you should use `buildsystem: autotools` and just have a `shell` source for the perl incantation. (or sed)

Reply:
done.

Reply:
I've added a comment to that effect in the manifest file.